assets/js/blocks/checkout/inner-blocks/checkout-billing-address-block/utils.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import {
+	DEFAULT_TITLE,
+	DEFAULT_DESCRIPTION,
+	DEFAULT_FORCED_BILLING_DESCRIPTION,
+	DEFAULT_FORCED_BILLING_TITLE,
+} from './constants';
+
+export const getBillingAddresssBlockTitle = (
+	title: string,
+	forcedBillingAddress: boolean
+): string => {
+	if ( forcedBillingAddress ) {
+		// returns default forced billing title when forced billing address is enabled and there is no title set
```

Comment:
Slight nit pick on these comments: please ensure we follow the WP Coding guidelines here: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#comments

Besides this, these new functions look good!